project.ts

```diff
@@ -312,8 +312,7 @@ const filters = {
     NewAccount: [],
   },
   transactionPayment: {
-    TransactionFeePaid: [],
-    FeeCharged: ['handleTransactionFeeCharged'],
+    TransactionFeePaid: ['handleTransactionFeeCharged'],
```

Comment:
Why not `FeeCharged` anymore?

Reply:
FeeCharged was never there in transactionPayment. When you refactored the project.ts it got messed up[ here](https://github.com/PolymeshAssociation/polymesh-subquery/pull/236/files#diff-45b7bc2074fd178435206ac01414cde32b1413b2e783a1c568a7ab6f4fbc14d0L362)